### PR TITLE
SYM-1942

### PR DESCRIPTION
**Pull request by rameezshaikh**
[russo] location city select nonfunctional

### CODE REVIEW

File: DataObjects/App.config [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <connectionStrings>
-    <add name="DSConnectionString" connectionString="Data Source=localhost;database=PlastEPortalMaster;User ID=ohum1;Password=Ohum@123;" providerName="System.Data.SqlClient" />
+    <add name="DSConnectionString" connectionString="Data Source=LPT0163\SA2014;database=PlastEPortalMaster;User ID=sa;Password=Ohum@123*;" providerName="System.Data.SqlClient" />
```
Comment (nickkarpuk) [inline DataObjects/App.config]: Your personal settings should be removed. Please merge latest from dev branch before you create a pull request and always review your changes before submitting them

File: SchedulerDataAccess/App.Config [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
  </configSections>
   <connectionStrings>
-    <add name="SchedulerEntities" connectionString="metadata=res://*/Model.SchedulerModel.csdl|res://*/Model.SchedulerModel.ssdl|res://*/Model.SchedulerModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=localhost;initial catalog=SchedulerRewrite;user id=ohum1;password=Ohum@123;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" />
+    <add name="SchedulerEntities" connectionString="metadata=res://*/Model.SchedulerModel.csdl|res://*/Model.SchedulerModel.ssdl|res://*/Model.SchedulerModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=LPT0163\SA2014;initial catalog=Scheduler_Rewrite;user id=sa;password=Ohum@123*;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" />
```
Comment (nickkarpuk) [inline SchedulerDataAccess/App.Config]: Your personal settings should be removed. Please merge latest from dev branch before you create a pull request and always review your changes before submitting them

File: SchedulerWebAPI/SchedulerWebAPI/Web.config [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
   </configSections>
   <connectionStrings>
-    <add name="SchedulerEntities" connectionString="metadata=res://*/Model.SchedulerModel.csdl|res://*/Model.SchedulerModel.ssdl|res://*/Model.SchedulerModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=localhost;initial catalog=SchedulerRewrite;user id=sa;password=Ohum@123*;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" />
+    <add name="SchedulerEntities" connectionString="metadata=res://*/Model.SchedulerModel.csdl|res://*/Model.SchedulerModel.ssdl|res://*/Model.SchedulerModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=LPT0163\SA2014;initial catalog=Scheduler_Rewrite;user id=sa;password=Ohum@123*;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" />
```
Comment (nickkarpuk) [inline SchedulerWebAPI/SchedulerWebAPI/Web.config]: Your personal settings should be removed. Please merge latest from dev branch before you create a pull request and always review your changes before submitting them

File: WCFService/Web.config [hunk 4]
```diff
@@ -50,7 +50,7 @@
     </exceptionPolicies>
   </exceptionHandling>
   <connectionStrings>
-    <add name="DSConnectionString" connectionString="Data Source=localhost;database=PlastEPortalMaster;User ID=ohum1;Password=Ohum@123;" providerName="System.Data.SqlClient" />
+    <add name="DSConnectionString" connectionString="Data Source=LPT0163\SA2014;database=PlastEPortalMaster;User ID=sa;Password=Ohum@123*;" providerName="System.Data.SqlClient" />
```
Comment (nickkarpuk) [inline WCFService/Web.config]: Your personal settings should be removed. Please merge latest from dev branch before you create a pull request and always review your changes before submitting them

File: WCFService/Web.config [hunk 5]
```diff
@@ -102,8 +102,8 @@
     <add key="DocumentEFaxStorageRootPathInServer" value="https://localhost/MainService/EFax/" />
     <add key="AssemblyVersion" value="1.0.2.8" />
     <add key="AssemblyFileVersion" value="1.0.2.8" />
-    <add key="ConnectionStringPart1" value="Data Source=localhost;Initial Catalog=" />
-    <add key="ConnectionStringPart2" value=";User ID=ohum1;Password=Ohum@123" />
+    <add key="ConnectionStringPart1" value="Data Source=LPT0163\SA2014;Initial Catalog=" />
+    <add key="ConnectionStringPart2" value=";User ID=sa;Password=Ohum@123*" />
```
Comment (nickkarpuk) [inline WCFService/Web.config]: Your personal settings should be removed. Please merge latest from dev branch before you create a pull request and always review your changes before submitting them

File: WebApplication/Areas/Scheduler/Content/scheduler.css [hunk 6]
```diff
@@ -8,6 +8,7 @@
         display: block;font-size: 12px;
     }
 
+
```
Comment (nickkarpuk) [inline WebApplication/Areas/Scheduler/Content/scheduler.css]: Is there a reason for a duplicate new line?

File: WebApplication/Web.config [hunk 7]
```diff
@@ -325,7 +325,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.1" />
```
Comment (nickkarpuk) [inline WebApplication/Web.config]: Please do not submit changes to web.config or app.config file unless they are required from functional point of view. Please merge latest from dev branch before you create a pull request and always review your changes before submitting them

File: WebApplication/WebApplication.csproj [hunk 8]
```diff
@@ -214,7 +214,7 @@
     <Reference Include="MoreLinq, Version=2.0.20029.0, Culture=neutral, PublicKeyToken=384d532d7e88985d, processorArchitecture=MSIL">
       <HintPath>..\packages\morelinq.2.2.0\lib\net35\MoreLinq.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=10.0.1.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```
Comment (nickkarpuk) [inline WebApplication/WebApplication.csproj]: Please do not submit changes to web.config or app.config file unless they are required from functional point of view. Please merge latest from dev branch before you create a pull request and always review your changes before submitting them

**Comment by rameezshaikh** [comment]
Giving multiple conflicts so clsoing this PR and will create new one

**Pull request closed without merging**